docker/amppkg.example.toml

```diff
@@ -0,0 +1,129 @@
+# Example toml using the fake testdata certs shipped with AMP Packager.
+#
+# The signed exchanges generated by this are NOT valid and will NOT work with
+# any browser or AMP cache!
+
+
+# This is a TOML 0.4.0 file, as specified by https://github.com/toml-lang/toml.
+
+# The port to listen on; 8080 is the default.
+# Port = 8080
+
+# Uncomment this line to serve only to localhost clients, e.g. for testing, by
+# binding on the loopback interface.
+# LocalOnly = true
+
+# The path to the PEM file containing the full certificate chain, ordered from
+# leaf to root. This will be served at /amppkg/cert/blahblahblah, where
+# "blahblahblah" is a stable unique identifier for the cert (currently, its
+# base64-encoded SHA-256).
+#
+# This certificate must use an EC P-256 key. (See https://goo.gl/pwK9EJ item
+# 3.1.5.) It must have at least one SCT, either as an X.509 extension or as an
+# extension to the OCSP responses from the URI mentioned in its Authority
+# Information Access extension. (See https://goo.gl/JQiyNs item 7.4.)
+#
+# To mitigate the risk of an attacker gaining access to your private key
+# through a vulnerability in this packager, you should generate a new
+# certificate/key pair for the domain(s) you wish to sign packages with. This
+# allows you to revoke that certificate without affecting your normal
+# web-serving traffic.
+CertFile = '/go/src/github.com/ampproject/amppackager/testdata/b1/fullchain.cert'
+
+# The path to the PEM file containing the private key that corresponds to the
+# leaf certificate in CertFile.
+KeyFile = '/go/src/github.com/ampproject/amppackager/testdata/b1/server.privkey'
+
+# The path to a file where the OCSP response will be cached. The parent
+# directory should exist, but the file need not. If this is a network-mounted
+# file, it should support shared/exclusive locking.
+OCSPCache = '/tmp/amppkg/ocsp'
```

Comment:
Maybe make this `/tmp/amppkg-ocsp` to save the need for a `mkdir` in your Dockerfile.

Reply:
done